uportal-war/src/main/java/org/springframework/orm/jpa/JpaAccessor.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2002-2012 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.orm.jpa;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.PersistenceException;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.dao.support.DataAccessUtils;
+
+/**
+ * Base class for JpaTemplate and JpaInterceptor, defining common
+ * properties such as EntityManagerFactory and flushing behavior.
+ *
+ * <p>Not intended to be used directly.
+ * See {@link JpaTemplate} and {@link JpaInterceptor}.
+ *
+ * @author Juergen Hoeller
+ * @since 2.0
+ * @see #setEntityManagerFactory
+ * @see #setEntityManager
+ * @see #setJpaDialect
+ * @see #setFlushEager
+ * @see JpaTemplate
+ * @see JpaInterceptor
+ * @see JpaDialect
+ * @deprecated as of Spring 3.1, in favor of native EntityManager usage
+ * (typically obtained through {@code @PersistenceContext})
```

Comment:
What would it take to *not* draw `JpaAccessor` into uPortal source control and instead follow this advice to use the native `EntityManager`?

Alternatively, if forking this deprecated code into the uPortal source tree is the way to go, could this class benefit from some comments right here about why this deprecated class is included in the uPortal codebase?